odh-dashboard/apps/habana/habana-quickstart.yaml

```
@@ -0,0 +1,239 @@
+            total = 0
+
+            with torch.no_grad():
+
+                for batch_idx, (data, targets) in enumerate(testloader):
+
+                    data, targets = data.to(device), targets.to(device)
+
+                    outputs = net(data)
+                    loss = criterion(outputs, targets)
+
+                    # API call to trigger execution
+
+                    htcore.mark_step()
+
+
+                    test_loss += loss.item()
+                    _, predicted = outputs.max(1)
+                    total += targets.size(0)
+                    correct += predicted.eq(targets).sum().item()
+
+            test_loss = test_loss/(batch_idx+1)
+            test_acc = 100.0*(correct/total)
+            print("Testing loss is {} and testing accuracy is {}".format(test_loss,test_acc))
+
+        def main():
+
+            epochs = 20
+            batch_size = 128
+            lr = 0.01
+            milestones = [10,15]
+            load_path = './data'
+            save_path = './checkpoints'
+
+            if(not os.path.exists(save_path)):
+                os.makedirs(save_path)
+            
+            # Target the Gaudi HPU device
+
+            device = torch.device("hpu")
+
+            
+            # Data
+            transform = transforms.Compose([
+                transforms.ToTensor(),
+            ])
+
+            trainset = torchvision.datasets.MNIST(root=load_path, train=True,
+                                                    download=True, transform=transform)
+            trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size,
+                                                    shuffle=True, num_workers=2)
+            testset = torchvision.datasets.MNIST(root=load_path, train=False,
+                                                download=True, transform=transform)
+            testloader = torch.utils.data.DataLoader(testset, batch_size=batch_size,
+                                                    shuffle=False, num_workers=2)
+
+            net = SimpleModel()
+            net.to(device)
+
+            criterion = nn.CrossEntropyLoss()
+            optimizer = optim.SGD(net.parameters(), lr=lr,
+                                momentum=0.9, weight_decay=5e-4)
+            scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer, milestones=milestones, gamma=0.1)
+
+            for epoch in range(1, epochs+1):
+                print("=====================================================================")
+                print("Epoch : {}".format(epoch))
+                train(net,criterion,optimizer,trainloader,device)
+                test(net,criterion,testloader,device)
+
+                torch.save(net.state_dict(), os.path.join(save_path,'epoch_{}.pth'.format(epoch)))
+
+                scheduler.step()
+
+        if __name__ == '__main__':
+            main()
+        ```
+      review:
+        failedTaskHelp: This task is not verified yet. Try the task again.
```

You still need the `instructions` attribute, try something like this:

```suggestion
        instructions: |-
          #### Verify the file is created:
          Do you have a file called `example.py`?
        failedTaskHelp: This task is not verified yet. Try the task again.
```